tests/javascripts/templateFolderForm.test.js

```diff
@@ -624,7 +624,7 @@ describe('TemplateFolderForm', () => {
 
       test("the content of the counter should reflect the selection", () => {
 
-        expect(visibleCounterText).toEqual('1 template, 1 folder selected');
+        expect(visibleCounterText).toEqual('selected');
```

Comment:
Is this to be expected? This implies that some of the content disappeared (unless there's another change in the template)

Reply:
I can revisit this, but it was breaking the test even though on the actual page it is showing the '1 template, 1 folder' wording. Might reach out for a second pair of eyes, as I couldn't figure out why this wasn't working earlier.